speaker/main.go

```diff
@@ -69,17 +70,19 @@ func main() {
 	}
 
 	var (
-		config      = flag.String("config", "config", "Kubernetes ConfigMap containing MetalLB's configuration")
+		configMap   = flag.String("config", "config", "Kubernetes ConfigMap containing MetalLB's configuration")
 		configNS    = flag.String("config-ns", "", "config file namespace (only needed when running outside of k8s)")
 		kubeconfig  = flag.String("kubeconfig", "", "absolute path to the kubeconfig file (only needed when running outside of k8s)")
-		host        = flag.String("host", os.Getenv("METALLB_HOST"), "HTTP host address")
+		metricsHost = flag.String("metrics-host", os.Getenv("METALLB_METRICS_HOST"), "Bind address for Prometheus metrics")
```

Comment:
I'm not sure it make sense to change the release notes in a feature PR. WDYT @rata?

Reply:
Yes, this makes sense in the "Action required" section of releases, IMHO. See here for examples: https://metallb.universe.tf/release-notes/

Reply:
You could add a new section for "TBD Next Release" where we collect notes for the next release

Reply:
I'm resolving this since at the moment we compile the release notes at release time. I agree we could improve the process but doing so isn't related to this specific PR.